pkg/sql/catalog/seqexpr/sequence.go

```diff
@@ -42,7 +42,10 @@ func (si *SeqIdentifier) IsByID() bool {
 // takes a sequence identifier as an arg (a sequence identifier can either be
 // a sequence name or an ID), wrapped in the SeqIdentifier type.
 // Returns the identifier of the sequence or nil if no sequence was found.
-func GetSequenceFromFunc(funcExpr *tree.FuncExpr) (*SeqIdentifier, error) {
+func GetSequenceFromFunc(
+	funcExpr *tree.FuncExpr,
+	getBuiltinProperties func(name string) (*tree.FunctionProperties, []tree.Overload),
+) (*SeqIdentifier, error) {
```

Comment:
nit: comment around these functions that this argument is commonly `builtins.GetBuiltinProperties`?